rotkehlchen/chain/evm/decoding/stakedao/decoder.py

```diff
@@ -0,0 +1,292 @@
+            msg_aggregator: 'MessagesAggregator',
+            claim_bribe_addresses: set['ChecksumEvmAddress'] | None = None,
+            claim_bounty_addresses: set['ChecksumEvmAddress'] | None = None,
+    ):
+        super().__init__(evm_inquirer, base_tools, msg_aggregator)
+        self.claim_bribe_addresses = claim_bribe_addresses
+        self.claim_bounty_addresses = claim_bounty_addresses
+        self.gauges: set[ChecksumEvmAddress] = set()
+
+    def reload_data(self) -> Mapping['ChecksumEvmAddress', tuple[Any, ...]] | None:
+        """Check that cache is up to date and refresh cache from db.
+        Returns a fresh addresses to decoders mapping.
+        """
+        if should_update_protocol_cache(
+            cache_key=CacheType.STAKEDAO_GAUGES,
+            args=(str(self.evm_inquirer.chain_id.value),),
+        ) is True:
+            query_stakedao_gauges()
+        if len(self.gauges) != 0:
+            return None  # we didn't update the globaldb cache, and we have the data already
+
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            self.gauges = set(globaldb_get_general_cache_values(  # type: ignore[arg-type]  # addresses are always checksummed
+                cursor=cursor,
+                key_parts=(CacheType.STAKEDAO_GAUGES, str(self.evm_inquirer.chain_id.value)),
+            ))
+
+        return self.addresses_to_decoders()
+
+    def _decode_claim(
+            self,
+            context: DecoderContext,
+            reward_token_address: ChecksumEvmAddress,
+            amount: int,
+            period: Timestamp,
+    ) -> DecodingOutput:
+        """Base functionality for claiming different types of stakedao votemarket bribes
+
+        Note: We don't check the user address in the logs as user is not always
+        the recipient according to the contract.
+        """
+        claimed_token = get_or_create_evm_token(
+            userdb=self.base.database,
+            evm_address=reward_token_address,
+            chain_id=self.evm_inquirer.chain_id,
+            evm_inquirer=self.evm_inquirer,
+        )
+        normalized_amount = token_normalized_value(amount, claimed_token)
+        for event in context.decoded_events:
+            if event.event_type == HistoryEventType.RECEIVE and event.event_subtype == HistoryEventSubType.NONE and event.asset == claimed_token and event.amount == normalized_amount:  # noqa: E501
+                event.event_subtype = HistoryEventSubType.REWARD
+                event.counterparty = CPT_STAKEDAO
+                event.notes = f'Claim {normalized_amount} {claimed_token.symbol} from StakeDAO veCRV bribes for the period starting at {timestamp_to_date(period, formatstr="%d/%m/%Y %H:%M:%S")}'  # noqa: E501
+                event.product = EvmProduct.BRIBE
+                break
+        else:  # not found
+            log.error(f'Stakedao bribe transfer was not found for {context.transaction.tx_hash.hex()}')  # noqa: E501
```

Comment:
This way also the chain will appear

```suggestion
            log.error(f'Stakedao bribe transfer was not found for {context.transaction}')  # noqa: E501
```